modules/apps/asset/asset-categories-service/src/main/java/com/liferay/asset/categories/internal/service/AssetEntryAssetCategoryRelAssetCategoryLocalServiceWrapper.java

```diff
@@ -111,10 +122,17 @@ public AssetCategory updateCategory(
 			categoryId);
 
 		if (!Objects.equals(category.getName(), name)) {
-			List<AssetEntry> assetEntries = _getAssetEntriesByAssetCategoryId(
-				category.getCategoryId());
+			TransactionCommitCallbackUtil.registerCallback(
+				() -> {
+					Message message = new Message();
+
+					message.put("categoryId", category.getCategoryId());
+
+					_messageBus.sendMessage(
+						DestinationNames.ASSET_CATEGORIES_REINDEX, message);
```

Comment:
DRY -> can you move this to a private method and reuse the code? reindexAssets or something like that?